requirements.txt

```diff
@@ -1,9 +1,11 @@
 django==1.8.2
-scipy==0.13.3
-matplotlib==1.3.1
-PyBrain==0.3
-psycopg2==2.6.1
-django-chartit==0.1
-simplejson==3.8.2
-Cython==0.23.5
-sklearn==0.0
+scipy
```

Comment:
apt-get deps are not necessary so please don't bother. It's built based on official Python 2.7 image, which is some sort of debian.